runtime/queries/nix/injections.scm

```diff
@@ -18,6 +17,16 @@
  (#set! injection.language "bash")
  (#set! injection.combined))
 
+; Common attribute keys corresponding to Python scripts,
+; such as those for NixOS VM tests in nixpkgs/nixos/tests.
+((binding
+   attrpath: (attrpath (identifier) @_path)
+   expression: (indented_string_expression
+     (string_fragment) @injection.content))
+ (#match? @_path "(^|\\.)testScript$")
+ (#set! injection.language "python")
+ (#set! injection.combined))
```

Comment:
This block should be moved on top of the pattern above it: the pattern above matches `.\wScript` and the stanzas that are higher in the file have higher precedence

Reply:
Applied.